frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -749,7 +750,10 @@ export const ClusterServiceVersionsPage: React.FC<ClusterServiceVersionsPageProp
     </Trans>
   );
 
-  const flatten = ({ clusterServiceVersions, subscriptions }) =>
+  const flatten: Flatten<{
+    clusterServiceVersions: K8sResourceKind[];
+    subscriptions: K8sResourceKind[];
```

Comment:
We should have specific types for these resource kinds.

Reply:
Used `ClusterServiceVersionKind` and `SubscriptionKind`. I had to add `status.currentCSV` to `SubscriptionKind` - verified on the actual resource, the field is really there